src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -11,135 +11,141 @@ namespace System.Text.RegularExpressions
     public partial class Capture
     {
         internal Capture() { }
-        public int Index { get { return default(int); } }
-        public int Length { get { return default(int); } }
-        public string Value { get { return default(string); } }
-        public override string ToString() { return default(string); }
+        public int Index { get { throw null; } }
```

Comment:
Is ```throw null``` our new approach for implementing member bodies in refs?  Seems fine (and likely less IL), we should just try to be consistent across the repo.  If this is our new approach, it'd be good to run a tool over all refs to do the full conversion.